src/frontend/views/status-guidance.njk

```diff
@@ -0,0 +1,34 @@
+{% set statusGuidance = "
```

Comment:
I think it would be nicer to do this as a macro

```
{% macro statusGuidance() %}
  <p class="govuk-body">
    Common banner states:
    <ul class="govuk-list govuk-list--bullet">
      <li>
      ...
```

it means you maintain the syntax highlightling. When you use it you need to add brackets, like you're invoking a function 

```
"content": {
  "html": statusGuidance() 
}
```

Reply:
Love it! All done.